src/biokbase/narrative/jobs/job.py

```diff
@@ -240,6 +240,28 @@ def was_terminal(self):
         else:
             return self._acc_state.get("status") in TERMINAL_STATUSES
 
+    @property
```

Comment:
I was uncertain about ACs so I added this unused method. Can delete

Reply:
Looking at the comments from the latest round of user testing, we may want to keep it, but it can be removed from this PR.